django/test/utils.py

```diff
@@ -836,6 +836,9 @@ def disable(self):
 def tag(*tags):
     """Decorator to add tags to a test class or method."""
     def decorator(obj):
-        setattr(obj, 'tags', set(tags))
+        if hasattr(obj, 'tags'):
+            obj.tags = obj.tags.union(tags)
```

Comment:
You can use [`set.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update) to avoid creating a new set instance in memory.